### ISSUE TITLE
[Clarification] Checking dataset quantity and heads up on future pull request

### ISSUE THREAD

**Issue by timothylimyl** [VainF/DeepLabV3Plus-Pytorch#58]
Hi @VainF ,

Can I check whether the cityscape dataset  that u have consist of 2975 images for training set and 500 for validation set? Checking on your dataset so that I can benchmark the metric accordingly, also how did u go about getting the GFLOPs?

I followed your scripts closely and added in hrnetv2 backbone without changing much of your code, I am currently training it and will soon request a pull if you are interested. I think hrnetv2 will be a very good backbone to integrate with deeplabv3plus.

**Comment by VainF** [comment]
Hi @timothylimyl, the standard train/val set, downloaded from https://www.cityscapes-dataset.com/downloads/. was used for in this work (2975 train + 500 val). The GFlops were obtained using https://github.com/Lyken17/pytorch-OpCounter.

It's would be great if you can add a new backbone to this repo. Feel free to request a new pull request for that! I will merge it as soon as possible.

**Comment by timothylimyl** [comment]
Hi @VainF, thanks for the info. I will ping you when it is done.

**Comment by timothylimyl** [comment]
@VainF ,

just to confirm, how did you train the other networks? 

was everything based on the default?

-> Scheduler: PolyLR
-> Optimiser: SGD
-> Loss: CrossEntropy
-> 30k Iterations on 16 batch size.

**Comment by timothylimyl** [comment]
Reopen for visibility.
